onnxruntime/test/perftest/ort_test_session.h

```diff
@@ -46,6 +48,7 @@ class OnnxRuntimeTestSession : public TestSession {
   std::vector<std::string> input_names_str_;
   const int input_length_;
   std::string provider_name_;
+  bool use_device_mem = false;
```

Comment:
Are we declaring this use_device_mem common for ORT perf test?

Reply:
common meaning for all EP's?
If this is the question then yes we are declaring use_device_mem common for all and initializing it with false.

If build is with OVEP and the user specifies to use this flag in the command line then only this flag will be true and which will create a buffer in NPU visible region.

Is there any way we can specify it just for OVEP?

Reply:
we should specify only for OVEP

Reply:
> common meaning for all EP's? If this is the question then yes we are declaring use_device_mem common for all and initializing it with false.
> 
> If build is with OVEP and the user specifies to use this flag in the command line then only this flag will be true and which will create a buffer in NPU visible region.
> 
> Is there any way we can specify it just for OVEP?

Okay understood. I confused it with -i option.